### PR TITLE
bugfix: consul not found tc cluster

### LINKED ISSUE #3772: seata 1.4.2 集成springboot项目 在启用consul配置中心的时候，项目无法完成启动，一直获取配置失败
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅱ. Describe what happened

 最近项目中考虑引入seata作为分布式事务解决方案，最开始我们是用springboot的yml配置文件作为配置中心，并做了压测，效果符合理想，后来参照官网高可用部署方式，我们用consul作为配置中心，就出现问题了，业务项目启动不起来，一直卡在读取配置，就像这样：

```
ERROR 45365 --- [           main] io.seata.config.ConfigFuture             : config operation timeout,cost:5000 ms,op:GET,dataId:transport.threadFactory.workerThreadPrefix
ERROR 45365 --- [           main] io.seata.config.ConfigFuture             : config operation timeout,cost:5003 ms,op:GET,dataId:transport.threadFactory.shareBossWorker
ERROR 45365 --- [           main] io.seata.config.ConfigFuture             : config operation timeout,cost:5000 ms,op:GET,dataId:transport.type
ERROR 45365 --- [           main] io.seata.config.ConfigFuture             : config operation timeout,cost:5001 ms,op:GET,dataId:transport.threadFactory.workerThreadSize
```
上面这个错误，断点调试 发现最开始是在SpringBootConfigurationProvider#get(dataId) 里面 获取 ConfigConsulProperties.class 这个bean 的时候卡住一段时间，大概有30秒，导致没有及时获取到consul连接配置，consul客户端没有及时完成初始化，这导致读取consul服务上配置数据超时失败，但是疑惑的是，后来consul客户端完成了初始化后，consul客户端从注册中心获取seata服务确总是失败，一直出现下面的错误，无法完成启动

```
ERROR 45365 --- [eoutChecker_2_1] i.s.c.r.netty.NettyClientChannelManager  : no available service found in cluster 'kun-ming', please make sure registry config correct and keep your seata server running
ERROR 45365 --- [eoutChecker_1_1] i.s.c.r.netty.NettyClientChannelManager  : no available service found in cluster 'kun-ming', please make sure registry config correct and keep your seata server running
```

我们事务组的配置如下：
service.vgroupMapping.ji9_tx_group = kun-ming

注册中心该服务其实是存在的

### Ⅲ. Describe what you expected to happen

我期望这个错误不应该存在，应该按照配置，能够在注册中心找到服务，期望这个问题尽快修复

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

最后我在ConsulRegisttryServiceImpl#refreshCluster(cluster) 方法中发现了问题，
refreshCluster(cluster) -调用-> getHealthyServices(getClusterName(), -1, -1)
其实getHealthService方法的service 参数应该使用上层方法refreshCluster(cluster) 传入的cluster 参数，而不应该从getClusterName() 获取服务名，因为getClusterName()获取的服务名是本地文件配置的。

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version : 1.8
- OS : centos 8 
- Others: 
 seata: seata-spring-boot-starter 1.4.2
springboot: 2.3.10.RELEASE
配置中心: consul
注册中心: consul

### PR DESCRIPTION
**Pull request by a364176773**
### Ⅱ. Does this pull request fix one issue?

fixes #3772

### CODE REVIEW

File: discovery/seata-discovery-consul/src/main/java/io/seata/discovery/registry/consul/ConsulRegistryServiceImpl.java [hunk 1]
```diff
@@ -268,10 +268,10 @@ private NewService createService(InetSocketAddress address) {
      * @param cluster
      */
     private void refreshCluster(String cluster) {
-        if (cluster == null) {
+        if (StringUtils.isBlank(cluster)) {
             return;
         }
-        Response<List<HealthService>> response = getHealthyServices(getClusterName(), -1, -1);
+        Response<List<HealthService>> response = getHealthyServices(cluster, -1, -1);
```
Comment (slievrly) [inline discovery/seata-discovery-consul/src/main/java/io/seata/discovery/registry/consul/ConsulRegistryServiceImpl.java:274]: add some comment `getClusterName` only for server

**Review by slievrly — APPROVED** [review]
LGTM

File: discovery/seata-discovery-consul/src/main/java/io/seata/discovery/registry/consul/ConsulRegistryServiceImpl.java (hunk 1 above)
Comment (a364176773) [reply discovery/seata-discovery-consul/src/main/java/io/seata/discovery/registry/consul/ConsulRegistryServiceImpl.java:274]: done

**Review by wangliang181230 — APPROVED** [review]
LGTM

**Review by xingfudeshi — APPROVED** [review]
LGTM.

**Pull request merged**